openapi-contracts/src/test/java/com/sinch/sdk/domains/voice/models/dto/svaml/ActionRunMenuDtoTest.java

```diff
@@ -34,7 +34,8 @@ public class ActionRunMenuDtoTest extends BaseTest {
                       .timeoutMills(500)
                       .maxTimeoutMills(123456)
                       .options(
-                          Collections.singletonList(new OptionDto().dtmf("#w").action("menu")))));
+                          Collections.singletonList(
+                              new OptionDto().dtmf("#w").action("menu(foo)")))));
```

Comment:
In the case of an option, I'm not sure `#w` is a valid dtmf as it's intended to be pressed by the user

Reply:
Yes, fFor this test, only providing a dtmf value with authorized characters was the only purpose